README.md

```diff
@@ -4,13 +4,59 @@
+- `parameter`: [variable](#parameter), see [hindcast input for the different models](#hindcast-input)
+- `origin`: name of the model [`ecmwf`, `eccc`, `ncep`] or modelling center [`ecmf`, `cwao`, `kwbc`]. Provide `origin` only for `training/test-input`/`hindcast/forecast-input`.
+- `date`: `YYYYMMDD` is the date of the 2020 forecast for `test-input`/`forecast-input`. The same dates are required for the on-the-fly `training-input`/`hindcast-input` but returns the multi-year hindcast for the given `MMDD` date. Please provide `int`, `str`, `np.datetime` or list of the former in format `YYYYMMDDD`. Providing no `date` keyword downloads all dates.
+- `format`: data format, choose from [`netcdf` (always available), `grb` (only for `input-*`), `zarr` (experimental)]
+
+## Coordinates
+
+Overview of the time-related coordinates.
+
+| `name` | CF convention `standard_name` | description | comment |
+| --- | --- | --- | --- |
+| `forecast_time` | forecast_reference_time | The forecast reference time in NWP is the "data time", the time of the analysis from which the forecast was made. It is not the time for which the forecast is valid. | |
+| `lead_time` | forecast_period | Forecast period is the time interval between the forecast reference time and the validity time. |  |
+| `valid_time` | time | time for which the forecast is valid | `forecast_time` + `lead_time` |
+
+All datasets are on a global 1.5 degree grid.
+
+
+## Parameter
+
+`parameter` describes the variable to download. The most important two variables for the `s2s-ai-challenge` are the two target variables `t2m` and `tp`:
+
+| paramter | long_name | standard_name | unit | description & aggregation type | week 3-4 aggregation | week 5-6 aggregation | link to source |
+| -------- | --------- | --- | --- | --- | --- | --- | --- |
+| `t2m`    | 2m temperature | air_temperature | K | Temperature at 2m height averaged for the date given | average [day 14, day 27] | average [day 28, day 41] | [model](https://confluence.ecmwf.int/display/S2S/S2S+Surface+Air+Temperature), [observations](http://iridl.ldeo.columbia.edu/SOURCES/.NOAA/.NCEP/.CPC/.temperature/.daily/)|
+| `tp`     | total precipitation | precipitation_amount | kg m-2 | Total precipitation accumulated from `forecast_time` until including `valid_time`, e.g. `lead_time = 1 days` accumulates precipitation_flux `pr` from 6-hourly steps 0,6,12,18 at date `forecast_time` | day 28 minus day 14 | day 42 minus day 28 | [model](https://confluence.ecmwf.int/display/S2S/S2S+Total+Precipitation) | 
+| `pr`     | precipitation flux | precipitation_flux | kg m-2 | Precipitation accumulated for the date given | use `tp` | use `tp` | [observations](http://iridl.ldeo.columbia.edu/SOURCES/.NOAA/.NCEP/.CPC/.UNIFIED_PRCP/.GAUGE_BASED/.GLOBAL/.v1p0/.extREALTIME/.rain) |
+
+Given the different nature of the parameters, `tp` `lead_time=1 days` decribes conditions for the same day as `t2m` `lead_time=0 days`.
```

Comment:
Please double-check with Frederic, or we could meet with him.
valid_time = forecast_time + lead_time.
This seems to mean that the tp and t2m data for the same value of valid_time do not refer to the physical state for same date, therefore are not valid for the same time.

Reply:
decribes -> describes

Reply:
I wrote Frederic: forecast_reference_time Jan 2nd 2020, tp day 1 and t2m day 0 describe both the weather of Jan 2nd. tp as aggregated pr from Jan 2nd 00:00 to Jan 3rd 00:00 and t2m as the average of Jan 2nd.

He confirmed: Yes, this is correct. 2mtm and tp are indeed different types of variables. So, it will be challenging to treat the verification of both variables the same way

Reply:
The sentence "Given the different nature of the parameters, `tp` `lead_time=1 days` decribes conditions for the same day as `t2m` `lead_time=0 days`." does not clarify, it only points to some tricky caveat but is not enough.

I would also add your nice example for jan 2nd.

Reply:
I do this in the renku datasets and inside the extra function. what do you mean by example? Written example or coding example?

Reply:
Actually, I would give the example for day 2 and day 3 to emphasize that tp is accumulated.